app/navigation/DocumentsNavigator.scala

```diff
@@ -86,13 +63,41 @@ class DocumentsNavigator @Inject() extends BaseNavigator {
       reviewRouteMap(page)(userAnswers)
   }
 
-  private def referenceAvailableRouting(mode: Mode = NormalMode): UserAnswers => Call = (userAnswers: UserAnswers) =>
-    userAnswers.get(ReferenceAvailablePage) match {
+  private def documentsCertificatesRouting(mode: Mode = NormalMode): UserAnswers => Call = (userAnswers: UserAnswers) =>
+    userAnswers.get(DocumentsCertificatesPage) match {
+      case Some(false) => routes.DocumentsCheckAnswersController.onPageLoad(userAnswers.ern, userAnswers.draftId)
+      case _ => userAnswers.get(DocumentsCount) match {
+        case Some(0) | None => routes.DocumentTypeController.onPageLoad(userAnswers.ern, userAnswers.draftId, 0, NormalMode)
```

Comment:
Wouldn't this be `1` ? To add the first one?

Reply:
If I'm understanding this correctly, 0 is the first one right?

Reply:
Ah yeah, missed that this gets implicitly converted into a Index model. 👍